### PR TITLE
fix following steps of install tidb-operator

### PR DESCRIPTION
**Pull request by weekface**
### What problem does this PR solve? 

This PR fixes: https://github.com/pingcap/tidb-operator/issues/2737

### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: charts/tidb-operator/templates/NOTES.txt [hunk 1]
```diff
@@ -1,7 +1,2 @@
-1. Make sure tidb-operator components are running
-   kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/instance={{ .Release.Name }}
-2. Install CRD
```
Comment (weekface) [inline charts/tidb-operator/templates/NOTES.txt:3]: When installing tidb-operator, CRD has already been installed in [the](https://pingcap.com/docs/tidb-in-kubernetes/stable/deploy-tidb-operator/#install-tidb-operator) [documents](https://pingcap.com/docs/tidb-in-kubernetes/stable/get-started/#deploy-tidb-operator).

File: charts/tidb-operator/templates/NOTES.txt [hunk 2]
```diff
@@ -1,7 +1,2 @@
...
-   kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml
-   kubectl get customresourcedefinitions
-3. Modify tidb-cluster/values.yaml and create a TiDB cluster by installing tidb-cluster charts
```
Comment (weekface) [inline charts/tidb-operator/templates/NOTES.txt:6]: Installation steps are different if user use different documents, user can follow guides in the document instead.

https://pingcap.com/docs/tidb-in-kubernetes/stable/get-started/#deploy-tidb-cluster
https://pingcap.com/docs/tidb-in-kubernetes/stable/deploy-on-general-kubernetes/

File: charts/tidb-operator/templates/NOTES.txt [hunk 3]
```diff
@@ -1,7 +1,5 @@
...
-   helm install tidb-cluster
+First, make sure tidb-operator components are running:
+
+    kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/instance={{ .Release.Name }}
+
+Then follow the next steps to create your TiDB clusters.
```
Comment (tennix) [inline charts/tidb-operator/templates/NOTES.txt:5]: What steps?

Comment (weekface) [reply charts/tidb-operator/templates/NOTES.txt:5]: These are the documents the user is reading. For example:

https://pingcap.com/docs/tidb-in-kubernetes/stable/get-started/#deploy-tidb-cluster
https://pingcap.com/docs/tidb-in-kubernetes/stable/deploy-on-general-kubernetes/

Comment (DanielZhangQD) [reply charts/tidb-operator/templates/NOTES.txt:5]: I think we can remove this, users can go to the next steps following the doc.

File: charts/tidb-operator/templates/NOTES.txt [hunk 4]
```diff
@@ -1,7 +1,5 @@
-1. Make sure tidb-operator components are running
-   kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/instance={{ .Release.Name }}
-2. Install CRD
-   kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml
-   kubectl get customresourcedefinitions
-3. Modify tidb-cluster/values.yaml and create a TiDB cluster by installing tidb-cluster charts
-   helm install tidb-cluster
+First, make sure tidb-operator components are running:
```
Comment (DanielZhangQD) [inline charts/tidb-operator/templates/NOTES.txt:1]:
```suggestion
Make sure tidb-operator components are running:
```

**Comment by weekface** [comment]
> Maybe we can add the [document link](https://pingcap.com/docs/tidb-in-kubernetes/stable/deploy-on-general-kubernetes/) in the note?

It is difficult to add these links because there are many `create TiDB cluster` guides in our documentation, and we cannot guess the documentation used by users, for example:

- https://pingcap.com/docs/tidb-in-kubernetes/stable/get-started/#deploy-tidb-cluster
- https://pingcap.com/docs/tidb-in-kubernetes/stable/deploy-on-general-kubernetes/
- https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.1/deploy-tidb-from-kubernetes-kind#%E7%AC%AC-3-%E6%AD%A5%E9%83%A8%E7%BD%B2-tidb-%E9%9B%86%E7%BE%A4
- https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.1/deploy-tidb-from-kubernetes-gke#%E9%83%A8%E7%BD%B2-tidb-%E9%9B%86%E7%BE%A4https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.1/deploy-tidb-from-kubernetes-minikube#%E9%83%A8%E7%BD%B2-tidb-%E9%9B%86%E7%BE%A4
- ...

**Pull request merged**